CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+Any contribution that you make to this repository will
```

Comment:
All of the previous code in this repository was BSD.  Since this PR isn't going to be substantially changing things, it needs to be under BSD as well so this should change to that.